ga4gh/cli.py

```diff
@@ -1741,7 +1748,7 @@ def addVariantSet(self):
             indexSuffix = ".tbi"
             # TODO support BCF input properly here by adding .csi
             indexFiles = [filename + indexSuffix for filename in dataUrls]
-
+        indexFiles = map(lambda x: os.path.abspath(x), indexFiles)
```

Comment:
You don't need the lambda function here, just use
```python
indexFiles = map(os.path.abspath, indexFiles)
```
There's a few more instances of this below too.

Nice use of functional tools though!